src/main/java/uk/gov/hmcts/reform/sscs/service/placeholders/FurtherEvidencePlaceholderService.java

```diff
@@ -128,6 +152,22 @@ private Address getJointPartyAddress(SscsCaseData caseData) {
             : ofNullable(caseData.getJointPartyAddress()).orElse(getEmptyAddress());
     }
 
+    private Address getOtherPartyAddress(SscsCaseData caseData, String otherPartyId) {
```

Comment:
This method is similar to https://github.com/hmcts/sscs-track-your-appeal-notifications/pull/1923/files#diff-2d4eb87c3d108969f4e7709adb6aef7f9b1426b2b8474b35a855eaec8eabe355R58-R68